core/src/avm2/object/responser_object.rs

```diff
@@ -0,0 +1,118 @@
+use crate::avm2::object::script_object::ScriptObjectData;
```

Comment:
This file is named `responser_object` - typo?

Reply:
Yep, thanks for spotting!